services/supervisor/src/mode.rs

```diff
@@ -103,3 +105,51 @@ impl Mode {
         }
     }
 }
+
+impl Default for Mode {
```

Comment:
I don't like it. `Simple` mode is not default by its nature, it's default within `exonum-cli` and nothing more than it. On the `Supervisor` level those modes do have equal rights.